FairBid/mintegral.plist.xml

```diff
@@ -84,6 +84,219 @@
+            </dict>
+              <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>3sh42y64q3.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>7UG5ZH24HU.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>ppxm28t8ap.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>wzmmz9fp6w.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>4w7y6s5ca2.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>9nlqeag3gk.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>4468km3ulz.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>p78axxw29g.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>g28c52eehv.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>5tjdwbrq8w.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>5lm9lj6jb7.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>YCLNXRL5PM.skadnetwork</string>
+            </dict>
+             <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>24t9a8vw3c.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>7rz58n8ntl.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>cg4yq2srnc.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>2U9PT9HC89.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>523jb4fst2.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>t38b2kh725.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>5a6flpkh64.skadnetwork</string>
+            </dict>
+            <dict>
+                <key>SKAdNetworkIdentifier</key>
+                <string>feyaarzu9v.skadnetwork</string>
+            </dict>
+
```

Comment:
I see on https://dev.mintegral.com/skadnetworkids.xml, there are total of 69 SKAdNetwork IDs for integral but according to this commit, we have less than 60. Maybe they added some during the night. Can you please add the missing networks ids?

Reply:
The reason is cus I removed a few duplications they had on their list.
Thx